.github/workflows/go.yml

```diff
@@ -2,9 +2,13 @@ name: Go
 
 on:
   push:
-    branches: [ main ]
+    branches: 
+      - main
+      - develope
```

Comment:
`develop` maybe?

Reply:
I can create another workflow, including only develop and adding the modifications there. thoughts?

Reply:
I forgot to add context, it says `develope` instead of `develop`: I believe we should change it to `develop` as it's our dev branch.

I don't think that creating another workflow will work for us, I think this is good as it is right now.

Reply:
Ohh yes! typo  :)